setup.py

```diff
@@ -5,19 +5,19 @@
 except ImportError:
     from distutils.core import setup
 
-__version__ = '0.0.2'
+__version__ = '0.0.3'
 
 packages = [
     'pylibmodbus',
 ]
 
 setup(
-    name='pylibmodbus',
+    name='pylibmodbus-mkat',
     version=__version__,
-    description="Python wrapper for libmodbus",
+    description="Private fork of Python wrapper for libmodbus (for KATCP/meerKAT support).",
```

Comment:
This repo isn't private, and not related to KATCP, so suggest:
"Python wrapper for libmodbus (fork for MeerKAT support).","